crates/sui-protocol-config/src/lib.rs

```diff
@@ -3375,6 +3376,19 @@ impl ProtocolConfig {
                     cfg.tx_context_ids_created_cost_base = Some(30);
                     cfg.tx_context_replace_cost_base = Some(30);
                     cfg.gas_model_version = Some(10);
+
+                    if chain != Chain::Mainnet {
+                        // Enable execution time estimate mode for congestion control on testnet.
+                        cfg.feature_flags.per_object_congestion_control_mode =
+                            PerObjectCongestionControlMode::ExecutionTimeEstimate(
+                                ExecutionTimeEstimateParams {
+                                    target_utilization: 30,
+                                    allowed_txn_cost_overage_burst_limit_us: 100_000, // 100 ms
+                                    randomness_scalar: 20,
```

Comment:
so this multiplies against target utilization meaning that target utilization for randomness txns will be 6%, right?

Reply:
Correct